README.md

```diff
@@ -45,13 +45,13 @@ S_05 = cov(method, X)
 
 In this section, `X` is the data matrix of size `n × p`, `S` is the sample covariance matrix with `S = κ (Xc' * Xc)` where `κ` is either `n` (uncorrected) or `n-1` (corrected) and `Xc` is the centred data matrix (see [docs](https://mateuszbaran.github.io/CovarianceEstimation.jl/dev)).
 
-* `Simple`: basic corrected and uncorrected sample covariance (via the `Statistics` package)
+* `SimpleCovariance`: basic corrected and uncorrected sample covariance (implemented in `StatsBase.jl`).
```

Comment:
maybe change _and_ in _or_, not sure (to imply that it's not both at once)

Reply:
OK, _or_ seems to be better.